governance/polls/meta/retroactive-poll-categorization.json

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
"340": ["Collateral"],
"341": ["Collateral"],
"342": ["Collateral"],
"343": ["Collateral"],
"344": ["Collateral"],
"345": ["Collateral"],
"346": ["Collateral"],
"347": ["Collateral"],
"348": ["Collateral"],
"349": ["Collateral"],
"350": ["Collateral"],
"351": ["Oracle"],
"352": ["Oracle"],
"353": ["Technical"],
"354": ["Governance"],
"355": ["Governance"],
"356": ["Collateral"],
"357": ["Governance"],
"358": ["Governance"],
"359": ["Risk Variable"],
"360": ["Risk Variable"],
"361": ["Risk Variable"],
"362": ["Risk Variable"],
"363": ["Risk Variable"],
"364": ["Risk Variable"],
"365": ["Risk Variable"],
"366": ["Technical"],
"367": ["Governance"],
"368": ["Governance"],
"369": ["Governance"],
"370": ["Technical"],
"371": ["Technical"],
"372": ["Governance"],
"373": ["Governance"],
"374": ["Governance"],
"375": ["Collateral"],
"376": ["Collateral"],
"377": ["Risk Variable"],
"378": ["Governance"],
"379": ["Risk Variable"],
"380": ["Risk Variable"],
"381": ["Technical"],
"382": ["Collateral"],
"383": ["Collateral"],
"384": ["Collateral"],
"385": ["Collateral"],
"386": ["Risk Variable"],
"387": ["Risk Variable"],
"388": ["Risk Variable"],
"389": ["Technical"],
"390": ["Collateral"],

Reply:
Added these to make the list up-to-date

Reply:
added these in 👍  now just need approval from somebody with the right permissions